raster/r.info/main.c

```diff
@@ -427,7 +427,7 @@ int main(int argc, char **argv)
 	    if (!gflag->answer) {
 		/* always report total number of cells */
 		fprintf(out, "cells=%jd\n",
-			(grass_int64)cellhd.rows * cellhd.cols);
+			(intmax_t)cellhd.rows * cellhd.cols);
```

Comment:
Please don't change `grass_int64`.

Reply:
Reverted to grass_int64. Changed format specifier, now using PRId64.